docs/content/en/docs/getting-started/install/_index.md

```diff
@@ -5,36 +5,38 @@ description: >
   Steps for setting up the Admin Machine
 ---
 
+{{% alert title="Warning" color="warning" %}}
+The Administrative machine should not be ephemeral. Artifacts created during cluster operations will be stored on the Administrative machine. Losing this machine could result in loss of cluster information.
+{{% /alert %}}
+
 EKS Anywhere will create and manage Kubernetes clusters on multiple providers.
 Currently we support creating development clusters locally using Docker and production clusters from providers listed on the [providers]({{< relref "/docs/getting-started/chooseprovider/" >}}) page.
 
-Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run Docker and add some binaries.
-From there, you create the cluster for your chosen provider.
+Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run all EKS Anywhere lifecycle operations as well as Docker, `kubectl` and the prerequisite utilites.
+From here you will need to install [`eksctl`](https://eksctl.io), a CLI tool for creating and managing clusters on EKS, and the [`eksctl-anywhere`](/docs/reference/eksctl/anywhere/) plugin, an extension to create and manage EKS Anywhere clusters on-premises, on your Administrative machine.
+You can then create a cluster for your chosen provider. 
 See [Create cluster workflow]({{< relref "../overview" >}}) for an overview of the cluster creation process.
 
-To create an EKS Anywhere cluster you will need [`eksctl`](https://eksctl.io) and the `eksctl-anywhere` plugin.
-This will let you create a cluster in multiple providers for local development or production workloads.
-
->**_NOTE:_** For Snow provider, the Snow devices will come with a pre-configured Admin AMI which can be used to create an Admin instance with all the necessary binaries, dependencies and artifacts to create an EKS Anywhere cluster. Skip the below steps and see [Create Snow production cluster]({{< relref "../snow" >}}) to get started with EKS Anywhere on Snow.
+>**_NOTE:_** For Snow provider, the Snow devices will come with a pre-configured Admin AMI which can be used to create an Admin instance with all the necessary binaries, dependencies and artifacts to create an EKS Anywhere cluster. Skip to the steps on [Create Snow production cluster]({{< relref "../snow/snow-getstarted" >}})to get started with EKS Anywhere on Snow.
 
 ### Administrative machine prerequisites
 
 #### System and network requirements
-- Mac OS 10.15 / Ubuntu 20.04.2 LTS
+- Mac OS 10.15+ / Ubuntu 20.04.2 LTS or 22.04 LTS
```

Comment:
@jaxesn do we want to add Rocky Linux and RHEL?

Reply:
Yeah lets do it

Reply:
Confirmed and updated to 8.8+